### ISSUE TITLE
June 2015 Info

### ISSUE THREAD

**Issue by jcblw** [cssla/cssla.github.io#3]
### Looking for talks

If you are interested in giving a talk please post your talk title and some information about the talk below. 

#### Date

This event I think should be slated for about a month from now on a tuesday. Lets say June 16th. Nothing is set in stone so if there is issues are you have ideas about scheduling please give feedback.

#### Venue

Im pretty sure I can get the Agent Ace in Santa Monica to host this on as well.

@itsjustmath you were interested in doing a talk about some utility stuff? Can you post some info.

:confetti_ball: Lets get this thing started :clap:

**Comment by itsjustmath** [comment]
Yeah sure. I'm wondering if it could be part of a broader talk on _ways of improving collaboration & communication b/w dev & design teams_. Or maybe I'm getting ahead of myself...

**Comment by jcblw** [comment]
I think we should start with some bite size talks @itsjustmath

**Comment by itsjustmath** [comment]
i think the single-purpose utilities talk would be a good one then.

**Comment by etanlubeck** [comment]
I'd like to talk about postcss

**Comment by etanlubeck** [comment]
Specifically migrating from SASS to postcss

**Comment by itsjustmath** [comment]
There's also [auto-enforceable coding guidelines](http://davidtheclark.com/scss-lint-styleguide/) with scss-lint

**Comment by danieljacobarcher** [comment]
@jcblw I'd be interested in talking. Maybe something like simple CSS tricks to implement responsive design into code, or Bourbon/Neat/Sass as a framework.

**Comment by AlecEmmons** [comment]
I want to come. I could talk about the history of print typography and how it correlates directly to what you are trying to achieve in CSS.
